src/sdk/pynni/tests/test_builtin_tuners.py

```diff
@@ -44,13 +66,20 @@ def setUp(self):
         self.params_each_round = 50
         self.exhaustive = False
 
+    def send_trial_callback(self, id_, params):
+        send(CommandType.NewTrialJob, _pack_parameter(id_, params))
+        
     def search_space_test_one(self, tuner_factory, search_space):
         tuner = tuner_factory()
         self.assertIsInstance(tuner, Tuner)
         tuner.update_search_space(search_space)
 
         for i in range(self.test_round):
-            parameters = tuner.generate_multiple_parameters(list(range(i * self.params_each_round,
+            if isinstance(tuner, PBTTuner):
+                parameters = tuner.generate_multiple_parameters(list(range(i * self.params_each_round,
+                                                                       (i + 1) * self.params_each_round)), st_callback=self.send_trial_callback)
+            else:
```

Comment:
I think `generate_multiple_parameters` of other tuners can be called with `st_callback` anyway?